### ISSUE TITLE
Destroy tag input fields

### ISSUE THREAD

**Issue by EvanHerman** [xoxco/jQuery-Tags-Input#189]
I am initializing the tag input fields via AJAX. I am unable to destroy the fields when the modal is closed. When the next modal is opened their are two tag input fields (the original + the new one).

Is there no way to destroy? such as 

`jQuery('input[name="video-tags"]').tagsInput('destroy');`

**Comment by EvanHerman** [comment]
closed but no response, that's professional. I'm going to venture a guess and assume the answer is no.

Answering someone when closing an issue is generally the way to go...just an FYI.

**Comment by lucknerjb** [comment]
@EvanHerman sorry, I did not mean to be unprofessional - you're right. Currently there is no way to destroy the plugin via an option but it will be possible in a few weeks when v2 comes out. I'll re-open the issue and keep you up to date through it.

**Comment by EvanHerman** [comment]
Thank you for the response, I appreciate that. 

No worries, I just assumed that you were closing the thread because you had no response and no idea. I know that others have asked this question as well, that is why I opened up an issue here. 

Again thank you for the response!

Evan

**Comment by lucknerjb** [comment]
No, that was not the reason at all :) I figured anything over 6 months the issue's author had either moved on or forgotten about it. I've been proved wrong!
